src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
+// three lines.
+/* jshint ignore: start */
+if ( typeof Symbol === "function" && Symbol.iterator ) {
```

Comment:
This, and no less, is what I currently consider to be test a valid "enough" Symbol polyfill: https://github.com/ljharb/object.assign/blob/master/hasSymbols.js

The "worst outcome" here could include a discoverable string property on the jQuery prototype, that people may rely on in browsers without native `Symbol`, and thus their code will break in newer browsers.

That said, every browser with native symbols also has `Symbol.iterator` so far, from my testing.